spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -251,11 +251,7 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 		}
 	}
 
-	/**
-	 * Return a copy of a given app definition where short form parameters have been expanded to their long form
-	 * (amongst the whitelisted supported properties of the app) if applicable.
-	 */
-	/*default*/ StreamAppDefinition qualifyParameters(StreamAppDefinition original, Resource resource) {
+	private Map<String, String> qualifyParameters(Map<String, String> appDeploymentProperties, Resource resource) {
```

Comment:
Since this method is used in two different invocations with `app properties defined at creation time` and `app properties defined at deployment time along with other deployment properties`, may be we can just call this as `properties` ?